github-for-jira.sd.yml

```diff
@@ -442,15 +453,6 @@ environmentOverrides:
 #          Period: 180
 #          ComparisonOperator: GreaterThanThreshold
 #          Statistic: Average
-        HighSeverityAlarmWhenTooManyBackend5xxErrors:
```

Comment:
Removing this per-env override, should remove the alert itself, because it already was removed globally